src/pages/student_council.tsx

```diff
@@ -0,0 +1,185 @@
+                                The council is responsible for representing the student body, organizing events, and
+                                communicating with the university&apos;s administration.
+                            </p>
+                        </div>
+                        <Image
+                            className="rounded-md md:ml-auto"
+                            src="/assets/images/student_council/group.jpg"
+                            alt="Student Council Group Photo"
+                            width={3641 / 11}
+                            height={3545 / 11}
+                        />
+                    </Paper>
+                </Grid.Col>
+                <Grid.Col span={{ base: 12, md: 6 }}>
+                    <Paper className="flex flex-col rounded-lg p-6 md:h-96 md:flex-row">
+                        <div>
+                            <h1 className="text-xl md:text-4xl">Upcoming Events</h1>
+                            <p className="w-96 text-base">
+                                The &quot;Welcome Freshmen&quot; event, orchestrated by the Student Council at Stamford
+                                International University, symbolizes the warm embrace awaiting new students. This event
+                                serves as an introduction to the vibrant community, showcasing the Council&rsquo;s
+                                dedication to fostering a welcoming environment and facilitating a smooth transition
+                                into university life.
+                            </p>
+                        </div>
+                        <Image
+                            className="rounded-md md:ml-auto"
+                            src="/assets/images/student_council/example.png"
+                            alt="Example Event Poster"
+                            width={3456 / 14}
+                            height={4608 / 14}
+                        />
+                    </Paper>
+                </Grid.Col>
+            </Grid>
+
+            <div className="pt-20">
+                <Carousel withIndicators
+                    slideSize="25%" height={200} slideGap="md" align="start" >
+                    {carouselSlides.map((slide, index) => (
+                        <Carousel.Slide key={index}>
+                            <Image src={slide.src} alt={slide.alt} className="w-full h-full object-cover" />
+                        </Carousel.Slide>
+                    ))}
+                </Carousel>
+            </div>
+
+            <div className="pt-10 flex flex-row items-center justify-center space-x-5">
+                <a href="https://www.instagram.com/studentcouncilstiu/" target="_blank">
+                    <UnstyledButton>
+                        <FaInstagram className="w-10 h-10" />
+                    </UnstyledButton>
+                </a>
```

Comment:
Keep in mind that since this is an external link, using an "a" element is fine, but if it is between routes, you have to use the Link component from NextJS to ensure that the page is pre-fetched.